### ISSUE TITLE
Video nonsense HTML code

### ISSUE THREAD

**Issue by sKopheK** [ahmedkaludi/accelerated-mobile-pages#4524]
Hi, upon inserting video block with video in WebM or MP4 format using APM Page Builder in WP, this comes out in HTML:

```
<div class="amp_mod ap_m_19 video"> 
    <amp-video class="" width="720" height="305" layout="responsive" autoplay controls>
        <source src="http://travel.lipe.cz/wp-content/uploads/2020/06/20191204_145944_1.webm" type="video/mp4">
        <source type="http://travel.lipe.cz/wp-content/uploads/2020/06/20191204_145944_1.webm" src="/video/tokyo.webm">
    </amp-video>
</div>
```

Setup:
![image](https://user-images.githubusercontent.com/5802917/83967972-4cef1180-a8e3-11ea-82a7-3105d7d191d1.png)

Could not find "tokyo" in any source file in the whole project of my WP page :)

See the source code in the actual page: http://travel.lipe.cz/2020/06/07/a-forgotten-empire/amp/

**Comment by ajeetku** [comment]
https://github.com/ahmedkaludi/accelerated-mobile-pages/commit/0fb2e310c069876d5ee50a782b132c1355961e83#diff-d09a7b96f0705e7784deebc14a9605a4R90 
here `<b>` used inside the esc_html, it will convert tags in &lt or &gt

**Comment by MohammedKaludi** [comment]
Ready to merge

**Comment by MohammedKaludi** [comment]
Fixed and pushed in 1.0.53.2

**Comment by sKopheK** [comment]
Thanks, markup looks better, but why does it make two `<source>` tags when only one source is provided? It does not distinguish what format is it at all. A viable way is via `src` parameter that does not require type specification, as shown in [documentation](https://amp.dev/documentation/components/amp-video/).

```
<amp-video
...
  src="https://yourhost.com/videos/myvideo.mp4"
  ...
>
</amp-video>
```
